packages/format-currency/src/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const FALLBACK_LOCALE = 'en';
+export const FALLBACK_CURRENCY = 'USD';
+export const GEO_LOCATION_ENDPOINT_URL = 'https://public-api.wordpress.com/geo/';
```

Comment:
This feels a bit external, it could be a parameter to `geolocateCurrencySymbol` but not sure if that makes sense outside of this API. It might be the case we don't export `geolocateCurrencySymbol` from one of the prospective packages and only keep it for WPCOM/Calypso.

Reply:
It's true that this is very WPCOM/calypso specific. It felt most useful to include it in the package when I implemented it for convenience and to simplify the package's API, but it's fine to externalize it if you have some way to override the currency symbol from outside the package (although `setCurrencySymbol` has now been removed so I'm not sure how best to do it).

Reply:
We'll keep it for now. My immediate guess is to keep this within i18n-calypso (or closer to wp-calypso) but not in the number-formatters package, which we may keep mostly generic.

I will return to this. Bringing back `setCurrenceSymbol` should be quick too 🤔